src/test/kotlin/sh/zachwal/button/presser/PresserManagerTest.kt

```diff
@@ -0,0 +1,45 @@
+package sh.zachwal.button.presser
+
+import io.mockk.mockk
+import kotlinx.coroutines.async
+import kotlinx.coroutines.newFixedThreadPoolContext
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withContext
+import org.junit.jupiter.api.Test
+
+internal class PresserManagerTest {
+
+    @Test
+    fun concurrentOperationsTest() {
```

Comment:
should there be some assertions in here? or does this just test that no exn is thrown lol

Reply:
no. yes.